Reader.go

```diff
@@ -49,6 +50,10 @@ func NewReader(r io.Reader, opt *ReaderOptions) (*Reader, error) {
 		w:             nil,
 	}
 
+	if opt.ContentTypes != nil {
```

Comment:
```suggestion
	if len(opt.ContentTypes) > 0 {
```
otherwise can panic if non-nil but len() == 0